CONTRIBUTING.md

```diff
@@ -96,48 +96,47 @@ If you have any further questions, then please join our [Discord Support Server]
+* Methods that override a method must be annotated with `@Override`.
+* Interfaces with only one method should be annotated using `@FunctionalInterface`.
+* If you deprecate a method, add an `@Deprecated` anotation.
 #### 3. Documentation
 * Every class and every public method should have a Javadocs section assigned to it.
 * New packages should have a `package-info.java` file with documentation about the package.
-* Classes should have an `@author` tag.
 * If there are any other relevant classes related to yours, add them using the `@see` tag.
+* If you deprecate a method, add an `@deprecated` section to the javadocs explaining why you did it.
 #### 4. Unit Tests
 * Try to write Unit Tests where possible.
 * Unit Test classes and methods should have no access modifier, not `public`, `protected` nor `private`.
-* Each Test should have a plain text `@DisplayName` annotation!
 #### 5. General best-practices
-* Do not use `Collection#forEach(x -> ...)`, use a proper `for (...)` loop!
-* Do not create new `Random` objects, use `ThreadLocalRandom.current()` instead!
-* Always declare Maps or Collections using their base type! (e.g. `List<String> list = new ArrayList<>();`)
-* When doing String operations like `String#toUppercase()`, always specify `Locale.ROOT` as an argument!
-* When reading or writing files, always specify the encoding using `StandardCharsets.UTF_8`!
-* Do not declare multiple fields/variables on the same line! (e.g. Don't do this: `int x, y, z;`)
-* Use a Logger, try to avoid `System.out.println(...)` and `Throwable#printStacktrace()`, use `Logger#log` instead!
+* Try to not use `Collection#forEach(x -> ...)`, use a proper `for (...)` loop.
+* Do not create new `Random` objects, use `ThreadLocalRandom.current()` instead.
+* Always declare Maps or Collections using their base type. (e.g. `List<String> list = new ArrayList<>();`)
+* When doing String operations like `String#toUppercase()`, always specify `Locale.ROOT` as an argument.
+* When reading or writing files, always specify the encoding using `StandardCharsets.UTF_8`.
+* Do not declare multiple fields/variables on the same line. (e.g. Don't do this: `int x, y, z;`)
+* Use a Logger, try to avoid `System.out.println(...)` and `Throwable#printStacktrace()`, use `Logger#log` instead.
 * Do not use Exceptions to validate data, empty catch blocks are a very bad practice, use other means like a regular expression to validate data.
 * If a parameter is annotated with `@Nonnull`, you should enforce this behaviour by doing `Validate.notNull(variable, "...");` and give a meaningful message about what went wrong
 * Any `switch/case` should always have a `default:` case at the end.
 * If you are working with a resource that must be closed, use a `try/with-resource`, this will automatically close the resource at the end. (e.g. `try (InputStream stream = ...) {`)
```

Comment:
```suggestion
* If you are working with a resource that must be closed, use a try-with-resources, this will automatically close the resource at the end (e.g. `try (InputStream stream = ...) {`). 
```